wurst/util/Illusion.wurst

```diff
@@ -0,0 +1,102 @@
+package Illusion
+// Credits goes to Flux for original vJass system, Trokkin for Wurst transition
+import NoWurst
+import Trigger
+import Unit
+import Orders
+import HashMap
+import DamageModify
+import Printing
+import OnUnitEnterLeave
+import AbilityObjEditing
+import InstantDummyCaster
+
+@configurable constant int ILLUSION_ID = 'AILS'
```

Comment:
What's the value in using a configurable constant instead of compiletime generated value?